EIPS/eip-9042.md

```diff
@@ -0,0 +1,97 @@
+---
+eip: 9042
```

Comment:
```suggestion
eip: 7819
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.